samples/outlook/85-tokens-and-service-calls/get-icaluid-as-organizer.yaml

```
@@ -0,0 +1,85 @@
+order: 8
+id: outlook-tokens-and-service-calls-get-icaluid-as-organizer
+name: Get an appointment's iCalUId as an organizer
+description: Uses EWS to get an appointment's iCalUId value where the user is the organizer.
+host: OUTLOOK
+api_set:
+    Mailbox: '1.3'
+script:
+    content: |
+        $("#run").click(run);
+
+        function run() {
+          Office.context.mailbox.item.saveAsync((result) => {
+            if (result.status === Office.AsyncResultStatus.Failed) {
+              console.log(result.error.message);
```

Consider using console.error so it will log it with error coloring and such...here and other instances
```suggestion
              console.error(result.error.message);
```